### PR TITLE
Added requested Mac x64/arm64 instructions.

### PR DESCRIPTION
**Pull request by MaanavD**
### Description

Added newly introduced API combinations to getting-started table.

### Motivation and Context
Attempting to get merged before / in time for ORT 1.17 release

### CODE REVIEW

File: src/routes/getting-started/table.svelte [hunk 1]
```diff
@@ -158,6 +158,24 @@
 
 		'mac,Python,X64,DefaultCPU': 'pip install onnxruntime',
 
+		'mac,Python,X64,CoreML': 'pip install onnxruntime',
+
+		'mac,Python,ARM64,CoreML': 'pip install onnxruntime',
```
[the remark this answers is not in the document]
Comment (edgchen1) [reply src/routes/getting-started/table.svelte:163]: oh, it's in a different place in this file. looks like the ARM64 entries were grouped together. maybe these newly added ones should go in the same place.

File: src/routes/getting-started/table.svelte [hunk 2]
```diff
@@ -143,11 +143,11 @@
 			"Install Nuget package&nbsp;<a class='text-blue-500' href='https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime' target='_blank'>Microsoft.ML.OnnxRuntime</a>",
 
 		'mac,C-API,X64,DefaultCPU':
-			"Download .tgz file from&nbsp;<a class='text-blue-500' href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>",
+			"Add 'onnxruntime-c' using CocoaPods or download the .tgz file from&nbsp;<a class='text-blue-500' href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>.",
 
 		'mac,C++,X64,DefaultCPU':
-			"Download .tgz file from&nbsp;<a class='text-blue-500' href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>",
-
+		"Add 'onnxruntime-c' using CocoaPods and refer to the&nbsp;<a class='text-blue-500' href='https://onnxruntime.ai/docs/tutorials/mobile/' target='_blank'>mobile deployment guide</a> or download .tgz file from&nbsp;<a class='text-blue-500' href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>.",
```
Comment (YUNQIUGUO) [inline src/routes/getting-started/table.svelte:149]: here, we also want to update and not including the mobile tutorial right? according to Ed's suggestion.

File: src/routes/getting-started/table.svelte [hunk 3]
```diff
@@ -158,6 +158,24 @@
 
 		'mac,Python,X64,DefaultCPU': 'pip install onnxruntime',
 
+		'mac,Python,X64,CoreML': 'pip install onnxruntime',
+
+		'mac,Python,ARM64,CoreML': 'pip install onnxruntime',
+
+		'mac,JS,ARM64,CoreML': 'npm install onnxruntime-react-native',
```
Comment (edgchen1) [inline src/routes/getting-started/table.svelte:165]: should it be onnxruntime-node?
https://github.com/microsoft/onnxruntime/blob/73356fc1920edd8df13ed62ac169de5a78923ee8/src/routes/getting-started/table.svelte#L313

does the JS package have the CoreML EP enabled? I'm not familiar with it.

Comment (YUNQIUGUO) [reply src/routes/getting-started/table.svelte:165]: https://github.com/microsoft/onnxruntime/blob/df740d7d152dfd04d29f4310a02d41b68f010b37/js/node/src/session_options_helper.cc#L80-L84

from here, looks like CoreML EP is enabled.

Comment (YUNQIUGUO) [reply src/routes/getting-started/table.svelte:165]: confirmed that the support with the combination here about "JS, CoreML" is not there yet. so maybe we can remove this line here for now.
```
'mac,JS,ARM64,CoreML': 'npm install onnxruntime-react-native',
```

**Review by YUNQIUGUO — APPROVED** [review]

**Comment by MaanavD** [comment]
https://maanavd.github.io/onnxruntime/

^Live preview

**Review by YUNQIUGUO — APPROVED** [review]

**Pull request merged**